pkg/runtime/runtime.go

```diff
@@ -1612,16 +1639,20 @@ func (a *DaprRuntime) publishMessageGRPC(ctx context.Context, msg *pubsubSubscri
 	case runtimev1pb.TopicEventResponse_SUCCESS:
 		// on uninitialized status, this is the case it defaults to as an uninitialized status defaults to 0 which is
 		// success from protobuf definition
+		diag.DefaultComponentMonitoring.PubsubIngressEvent(ctx, msg.pubsub, string(pubsub.Success), msg.topic, elapsed)
```

Comment:
lowercase for `success`

Reply:
1. These values are taken from Components Contrib
2. This is not how Golang works. Lowercasing them would mean these fields are unexported, and thus unavailable to `runtime.go`.